modules/homePage/component/v2/FourthJumbotron/FourthJumbotronComponent.tsx

```diff
@@ -1,7 +1,7 @@
 import ButtonPrimary from "../../../../../uikit/buttonV2/ButtonPrimary";
 
 function ForthJumobtronBox({ children, className }: { className?: string; children?: React.ReactNode }) {
-    return <div className={`relative mt-4 h-[236px] max-w-full overflow-hidden rounded-[36px] bg-dark-background-elevated pt-8 pl-8 text-left sm:h-[280px] xl:h-[300px]     ${className || ""}`}>{children}</div>;
+    return <div className={`relative mt-4 h-[236px] max-w-full overflow-hidden rounded-3xl bg-dark-background-elevated pt-8 pl-8 text-left sm:h-[280px] xl:h-[300px]     ${className || ""}`}>{children}</div>;
```

Comment:
1. I think `mt-4` is not necessary
2. Use `h-[236px]` (height in mobile)
3. Use `sm:h-[314px]` (height in any other screen)
4. Remove other height configuration in box

Reply:
hmm i already used that configuration 
but some how in sm, IMO while we're using 314, it's not proportional